home-excercises-05-2022/src/main/java/io/codelex/arrays/practice/exercise7/TicTacToe.java

```diff
@@ -0,0 +1,92 @@
+package io.codelex.arrays.practice.exercise7;
+
+import java.util.Scanner;
+
+public class TicTacToe {
+
+    private static char[][] board = new char[3][3];
+
+    public static void main(String[] args) {
+        Scanner keyboard = new Scanner(System.in);
+        char turn = 'X';
+        char winner = ' ';
+        int row = -1;
+        int column = -1;
+        int turnCount = 0;
+        initBoard();
+        while (winner == ' ') {
+            displayBoard();
+            if (turnCount == 9) {
+                System.out.println("The game is a tie.");
+                return;
+            }
+            System.out.print(turn + ", choose your location (row, column): ");
+            while (true) {
+                row = keyboard.nextInt();
+                column = keyboard.nextInt();
+                if (row < 0 || row > 2 || column < 0 || column > 2 || board[row][column] != ' ') {
```

Comment:
kad parādās tik daudz conditions, var apsvērt iznest uz palīgmetodi, būs lasāmāk
`is ValidLocation()` piemēŗam

Reply:
izveidoju papildus metodi